src/frontend/planner_test/tests/testdata/nexmark.yaml

```diff
@@ -303,7 +303,7 @@
             └─StreamHashAgg { group_key: [window_start], aggs: [count, max(count)] }
               └─StreamExchange { dist: HashShard(window_start) }
                 └─StreamProject { exprs: [bid.auction, window_start, count] }
-                  └─StreamShare { id = 956 }
+                  └─StreamShare { id = 985 }
```

Comment:
The two `StreamProject` around this line seems to be the same, are they expected to be fixed in this PR?

Reply:
This project is necessary and expected because `StreamShare` eventually will be converted into an exchange. however two exchange connected directly is unsupported, so we will insert an identity project between the share operator and the above exchange for now.